Jetchat/app/src/main/java/com/example/compose/jetchat/widget/composables/MessagesWidget.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.compose.jetchat.widget.composables
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.unit.dp
+import androidx.glance.GlanceModifier
+import androidx.glance.ImageProvider
+import androidx.glance.LocalContext
+import androidx.glance.action.actionStartActivity
+import androidx.glance.action.clickable
+import androidx.glance.appwidget.components.Scaffold
+import androidx.glance.appwidget.components.TitleBar
+import androidx.glance.appwidget.lazy.LazyColumn
+import androidx.glance.layout.Column
+import androidx.glance.layout.Spacer
+import androidx.glance.layout.fillMaxWidth
+import androidx.glance.layout.height
+import androidx.glance.text.FontWeight
+import androidx.glance.text.Text
+import androidx.glance.text.TextStyle
+import com.example.compose.jetchat.NavActivity
+import com.example.compose.jetchat.R
+import com.example.compose.jetchat.conversation.Message
+import com.example.compose.jetchat.data.unreadMessages
+
+@Composable
+fun MessagesWidget() {
+    Scaffold(titleBar = {
+        TitleBar(
+            startIcon = ImageProvider(R.drawable.ic_jetchat),
+            iconColor = null,
+            title = LocalContext.current.getString(R.string.app_name_unreads),
+        )
+    }) {
+        val unreadMessage = unreadMessages
+        LazyColumn(modifier = GlanceModifier.fillMaxWidth()) {
+            unreadMessage.forEach {
+                item {
+                    Column(modifier = GlanceModifier.fillMaxWidth()) {
+                        MessageItem(it)
+                        Spacer(modifier = GlanceModifier.height(10.dp))
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MessageItem(message: Message) {
+    Column(modifier = GlanceModifier.clickable(actionStartActivity<NavActivity>()).fillMaxWidth()) {
+        Text(
+            text = message.author,
+            style = TextStyle(fontWeight = FontWeight.Bold)
+        )
+        Text(
+            text = message.content,
```

Comment:
I'm not a fan of us not being able to format the text in any form. Showing the unformatted font looks unstyled :(